.github/workflows/functional-test.yml

```diff
@@ -67,15 +71,17 @@ jobs:
         script: |
           npm run e2e-test:driver
           npm run e2e-test:commands
-          # skip find tests with api-25 as they are too flaky 
-          [[ "${{ matrix.apiLevel }}" == "25" ]] || npm run e2e-test:commands:find
+          npm run e2e-test:commands:find
           npm run e2e-test:commands:general
           npm run e2e-test:commands:keyboard
         avd-name: ${{ env.ANDROID_AVD }}
         sdcard-path-or-size: 1500M
         api-level: ${{ matrix.apiLevel }}
         disable-spellchecker: true
         target: ${{ matrix.emuTag }}
+        arch: ${{ matrix.arch }}
+        ram-size: 4096M
+        heap-size: 1024M
```

Comment:
macos-12 has 14G RAM
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources